esi-custom.yaml

```diff
@@ -1,4 +1,29 @@
+resource_registry:
+  OS::TripleO::Services::NovaCompute: OS::Heat::None
+  OS::TripleO::Services::NovaLibvirt: OS::Heat::None
+  OS::TripleO::Services::Redis: /usr/share/openstack-tripleo-heat-templates/deployment/database/redis-pacemaker-puppet.yaml
+
 parameter_defaults:
+  NeutronBridgeMappings: datacentre:br-ctlplane
+  NeutronPhysicalBridge: br-ctlplane
+  NeutronFlatNetworks: datacentre
+  NeutronMechanismDrivers: [openvswitch, baremetal]
+  NeutronNetworkVLANRanges: datacentre:611:620
+  NeutronTypeDrivers: [local, geneve, vlan, flat, vxlan]
+
+  NovaLibvirtVolumeUseMultipath:  true
+  NovaComputeOptVolumes:
+    - /etc/multipath.conf:/etc/multipath.conf:ro
+    - /etc/multipath/:/etc/multipath/:rw
+  CinderVolumeOptVolumes:
+    - /etc/multipath.conf:/etc/multipath.conf:ro
+    - /etc/multipath/:/etc/multipath/:rw
+
+  # configure neutron to always provide routes to the metadata
+  # service via dhcp
+  NeutronEnableForceMetadata: true
+  SELinuxMode: permissive
```

Comment:
curious why we need to set this to permissive?

Reply:
We maybe don't; it's just a setting on the demo server while I was testing things out. Danni, take it out for now and see how it goes!

Reply:
Ah, I copied it from esi-sanbox/standalone_parameters.yaml in the demo server. Haven't tested it yet. I didn't set it to permissive in the previous deployment and didn't see any issue related with SELinuxMode.